service/framework.go

```diff
@@ -241,6 +242,10 @@ func newCRDFramework(config Config) (*framework.Framework, error) {
 		}
 	}
 
+	versionedResources := map[string][]framework.Resource{
+		keyv2.K8sCloudConfig_V_2_0_0: resources,
```

Comment:
Here you say the resource router should route to a set of resources based on the k8scloudconfig version. Resources of the operatorkit framework do not have to do anything with k8scloudconfig. These are two different concepts. Operatorkit is open and abstracted. k8scloudconfig is an implentation detail within Giant Swarm. The resource router should route resources based on version bundle versions the operator provides.